docs/studio/getting-started-with-studio.md

```diff
@@ -0,0 +1,200 @@
+| Orientation Marker Corner | Upper Right, Upper Left, Lower Left, Lower Right |
+| PCA Range | Number of standard deviations for PCA slider |
+| Mesh Cache Enabled | Whether to use a cache of meshes |
+| Mesh Cache Epsilon | Determination of how much different a mesh must be before it is considered a different mesh, for caching purposes |
+| Mesh Cache Memory to Use | Percentage of phyisical memory to allow mesh cache to use |
+| Mesh Parallel Reconstruction | Whether to use parallelized mesh reconstruction |
+| Mesh Parallel Number of Threads | How many threads to use for parallelized mesh reconstruction |
+| Log Location | A display of where the current log file is located |
+
+<br/>
+## Viewer Controls
+
+At the bottom of the viewers are the controls for the viewers:
+
+![ShapeWorks Studio Viewer Controls](../img/studio/studio_viewer_controls.png)
+
+Each button has tooltips with descriptions.  The controls from left to right are
+
+* Autoview / reset view
+* Glyph visibility with dropdown for glyph size and detail
+* Isosurface visibility
+* View mode (original/groomed/reconstructed)
+* Center (whether to center each object based on center of mass)
+* Number of views in lightbox (1,2,4,9,16,25,36,49,64)
+
+Additionally, when feature volumes or meshes with scalars are loaded, the extended panel will be displayed:
+
+![ShapeWorks Studio Viewer Extended Controls](../img/studio/studio_viewer_controls_extended.png)
+
+This adds the feature map / scalar selection and a checkbox to apply a uniform scale to all viewers, vs local scale for each.
+
+## Keyboard Shortcuts
+
+The keyboard shortcuts are available from the Help -> Keyboard Shortcuts menu item.
+
+![ShapeWorks Studio Keyboard Shortcuts](../img/studio/studio_keyboard_shortcuts.png)
+
+### Point selection 
+
+As shown in the keyboard shortcuts panel, you can hover over a particle and press the '1' key.  This will select this point on all viewers.
+
+The selected point will become white and all other points will change to a color based on the distance to the chosen point.
+
+![ShapeWorks Studio Particle Selection](../img/studio/studio_particle_selection.png)
+
+To turn off this display, press '1' while hovering over something other than a particle.
+
+Additionally, after selecting a point, you may select another point by pressing '2' while hovering.  This will set the first point to white, the second point to yellow and the rest to black.  This is an easy way to see how particlar particles vary over shapes.
+
+![ShapeWorks Studio Particle Selection2](../img/studio/studio_particle_selection2.png)
+
+
+## File menus
+
+Studio's file menu is shown below.
+
+![ShapeWorks Studio File Menu](../img/studio/studio_file_menu.png)
+
+* New Project - Create a new project
+* Open Project - Open a project from disk
+* Save Project - Save project to disk
+* Save Project As - Save a copy of the project to a new file
+* Import Images... - Import new shapes
```

Comment:
Does `Import Image` import segmentations or meshes? I think we should be more explicit as we will also import imaging data with shapes. If it can import both, let's use "Image Shapes ..."

Reply:
Yeah, I agree.